ddp-workspace/projects/ddp-cgc/src/app/constants/activity-status-code.ts

```diff
@@ -0,0 +1,5 @@
+export enum ActivityStatusCode {
```

Comment:
No need to create your own enum, you can just import it from sdk: `ddp-workspace/projects/ddp-sdk/src/lib/models/activity/activityStatusCodes.ts`

Reply:
Fixed